src/Symfony/Component/HttpKernel/Profiler/FileProfilerStorage.php

```diff
@@ -57,14 +58,18 @@ public function find(?string $ip, ?string $url, ?int $limit, ?string $method, in
         while (\count($result) < $limit && $line = $this->readLineFromFile($file)) {
             $values = str_getcsv($line);
 
-            if (7 !== \count($values)) {
+            if (8 !== \count($values)) {
```

Comment:
this invalidates existing profiles, maybe we can still accept them?

Reply:
Yes, it also made me realize that I did not use the expiration introduced since I've opened the PR.
Fixed.